demo/src/pages/notification/Notification.tsx

```diff
@@ -0,0 +1,96 @@
+import { IUseNotificationOptions } from "openfin-react-hooks";
+import * as Prism from "prismjs";
+import React, { useEffect, useState } from "react";
+import Demo from "../../common/Demo";
+import LaunchConfiguration from "../../common/LaunchConfiguration";
+import CodeExample from "./components/CodeExample";
+import DemoActions from "./components/DemoActions";
+import Checkboxes from "./components/inputs/Checkboxes";
+import TextInputs from "./components/inputs/TextInputs";
+import ILaunchConfig from "./interfaces/ILaunchConfig";
+import styles from "./Notification.module.css";
+
+const HTML_URL: string | undefined = process.env.REACT_APP_SAMPLE_WINDOW_HTML;
+const CSS_URL: string | undefined =
+    process.env.REACT_APP_SAMPLE_NOTIFICATION_CSS;
+const INITIAL_TEXT_AREA_VALUE: string = `<div class="App_containerApp__F0W0w">
+  <p>Hello World!</p>
+</div>`;
+const NOTIFICATION_BODY_AS_HOOK_OPTION = (
+    <div className="App_containerApp__F0W0w">
+        <p>
+            This jsx was passed in as a part of <code>NOTIFICATION_HOOK_OPTIONS</code>
+            . See "Code Example" Section for more information.
+        </p>
+    </div>
+);
+const INITIAL_NOTIFICATION_OPTIONS: ILaunchConfig = {
+    cssUrl: CSS_URL,
+    notificationOptions: {
+        message: null,
+        timeout: "never",
+        url: HTML_URL,
+    },
+    shouldInheritCss: true,
+    shouldInheritScripts: true,
+    shouldLoadJsxAfterLaunch: true,
+};
+
+const Notification: React.FC = () => {
```

Comment:
There's an issue here if i try and populate a dismissed notification / closed child window

Reply:
Very good point. I now added onDissmiss, onClose overrides that reset all state variables.